etc/ReplayOfflineConfiguration.py

```diff
@@ -491,7 +493,7 @@
                dqm_sequences=["@common"],
                tape_node="T1_FR_CCIN2P3_MSS",
                disk_node="T1_FR_CCIN2P3_Disk",
-               physics_skims=["EXODisplacedJet", "EXODelayedJet", "EXODTCluster", "LogError", "LogErrorMonitor"],
+               physics_skims=["EXODisplacedJet", "EXODelayedJet", "EXODTCluster", "LogError", "LogErrorMonitor", "EXOCSCCluster"],
                # EXOCSCCluster needs to be added when 12_4_4 is introduced
```

Comment:
in the next PR we should remember to remove this comment

Reply:
I will do it in my PR, i'm preparing it now

Reply:
Thanks Francesco!